imports/ui/JapaneseNewGame.html

```diff
@@ -301,7 +301,7 @@
 
 <template name="jpn_points">
 	<span class="player-select-header"><b>Points: </b></span>
-	<select name="points">
+	<select name="points" class="points">
```

Comment:
Can you check and see if name="points" and class="points" can be merged?  Seems like it might be unnecessary duplication.